docs/src/pages/components/timeline/timeline.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Timeline React component
+components: Timeline, TimelineItem, TimelineSeparator, TimelineDot, TimelineConnector, TimelineContent, TimelineOppositeContent
+---
+
+# Timeline
+
+<p class="description">The timeline displays a list of events in chronological order.</p>
+
+**Note:** This component is not documented in the [Material Design guidelines](https://material.io/), but Material-UI supports it.
+
+## Simple timeline
+
+Simeple timeline showing list of events.
```

Comment:
Sorry on phone - simple. But aren’t we moving away from describing things as simple?

Reply:
Hmm, how do we then describe the "default" component, without any props?

Reply:
Let's fix the typo first
```suggestion
Simple timeline showing list of events.
```

We should move away from "simple" anyway:

> What might be simple for you might not be simple for others. Try eliminating this word from the sentence because usually the same meaning can be conveyed without it.

-- https://developers.google.com/style/word-list#letter-s

But this is probably better discussed in a dedicated PR.

Reply:
Agree, we started using basic https://material-ui.com/components/pagination/#basic-pagination.

Reply:
cc @mbrookes https://developers.google.com/style/word-list looks like a great guideline, maybe we should put something like this together.

Reply:
Yep, I've been objecting to "simple" for quite a while now, so Pagination used "basic" instead to finally start the transition. Happy to see other similar changes applied if they make sense.